lib/vault/vault_setup.sh

```diff
@@ -167,3 +167,9 @@ if [ -f tmp/vault/service_stores.txt ]; then
         bash create_service_store.sh $service
     done <tmp/vault/service_stores.txt
 fi
+
+crontab -l | grep -q vault_setup
+if [[ $? -ne 0 ]]; then
+  echo "creating crontab"
+  echo "0 0 1 * * cd $PWD; bash vault_setup.sh" | crontab -
```

Comment:
This will overwrite everything else in the crontab; looking online for solutions they all recommend -l but that deletes all comments

Reply:
Fixed in new commit: I saved out the old crontab and appended to that one.